protos/flyteidl/core/interface.proto

```diff
@@ -16,10 +16,16 @@ message Variable {
     string description = 2;
 }
 
-// A map of Variables
+// A map of Variables in the form of repeated map entries
 message VariableMap {
     // Defines a map of variable names to variables.
-    map<string, Variable> variables = 1;
+    repeated VariableMapFieldEntry variables = 1;
```

Comment:
isnt this a breaking change?

Reply:
if not, so we realized that variables and parameters being separate was not a good idea, we should just have one - one with default values